Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.py

```diff
@@ -781,6 +1275,11 @@ def main():
             demisto.results(get_social_applications_command())
         elif demisto.command() == 'cortex-search-by-file-hash':
             demisto.results(search_by_file_hash_command())
+        elif demisto.command() == 'cortex-query-traffic-logs' \
```

Comment:
use `in` set/typle/list instead of 4 condition components

Reply:
changed it to a different condition for each command